app/src/main/java/com/nextcloud/talk/controllers/CallController.java

```diff
@@ -2044,11 +2050,11 @@ private void setupVideoStreamForLayout(@Nullable MediaStream mediaStream, String
             nick = getPeerConnectionWrapperForSessionIdAndType(session, videoStreamType, false).getNick();
         }
 
-        String userId;
+        String userId = "";
         if (hasMCU) {
             userId = webSocketClient.getUserIdForSession(session);
-        } else {
-            userId = participantMap.get(session).getUserId();
+        } else if (participantMap.get(session).getActorType() == Participant.ActorType.USERS) {
+            userId = participantMap.get(session).getActorId();
```

Comment:
is there a general fall back? because right now the userId can be an empty string if none of the ifs turns out to be true.

Reply:
before getUserId fell back and now there is a default value on the initialization